python/langsmith/client.py

```diff
@@ -2529,6 +2529,8 @@ def create_dataset(
         *,
         description: Optional[str] = None,
         data_type: ls_schemas.DataType = ls_schemas.DataType.kv,
+        inputs_schema_definition: Optional[Dict[str, Any]] = None,
```

Comment:
is "definition" necessary? My fingers hurt just looking at this

Reply:
I matched the API exactly. Do we mismatch ever otherwise?

Reply:
We do mismatch - our apis are pretty confusing sometimes